abilities/tools/abilityMaker/CMakeLists.txt

```diff
@@ -8,9 +8,13 @@ find_package(Qt5 COMPONENTS Quick REQUIRED)
 
 add_executable(AbilityMaker ${abilityMakerSources})
 
-target_compile_features(AbilityMaker PRIVATE cxx_std_17)
+target_compile_features(AbilityMaker PRIVATE cxx_std_20)
 target_compile_definitions(AbilityMaker
   PRIVATE $<$<OR:$<CONFIG:Debug>,$<CONFIG:RelWithDebInfo>>:QT_QML_DEBUG>)
+if (UNIX)
+  target_compile_options(AbilityMaker PRIVATE -fcoroutines)

```

Comment:
This tool is supposed to be just a small application for helping creating abilities. It's not supposed to work with coroutines, so I doubt this is needed.

Reply:
It doesn't build without coroutines.